config/locales/sc.yml

```diff
@@ -581,8 +669,14 @@ sc:
       more_details_html: Pro àteros detàllios, bide sa <a href="%{terms_path}">normativa de riservadesa</a>.
       username_available: Su nòmine de utente tuo at a torrare a èssere a disponimentu
       username_unavailable: Su nòmine de utente tuo no at a abarrare a disponimentu
+  disputes:
+    strikes:
+      title_actions:
+        none: Atentzione
   domain_validator:
     invalid_domain: no est unu nòmine de domìniu vàlidu
+  edit_profile:
+    other: Àteru
```

Comment:
I believe the error from `bundle exec i18n-tasks unused` is a false positive caused by `other` being the single sub-key and tripping the plural detection heuristic